spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8SecretsPropertySourceLocator.java

```diff
@@ -69,11 +68,10 @@ public Fabric8SecretsPropertySourceLocator(KubernetesClient client, SecretsConfi
 	@Override
 	protected MapPropertySource getPropertySource(ConfigurableEnvironment environment,
 			SecretsConfigProperties.NormalizedSource normalizedSource, String configurationTarget) {
-		String secretName = getApplicationName(environment, normalizedSource.getName(), configurationTarget);
```

Comment:
this has move to the caller. It does not really matter here, since `getApplicationName` is in commons _anyway_; but for the fix of the issue this matters a lot.